handler/handler.go

```diff
@@ -258,34 +267,29 @@ func (h *ReindexRequestedHandler) getAndSendDatasetURLsFromLatestMetadata(ctx co
 				}
 			}()
 		}
-		log.Info(ctx, "successfully handled metadata")
 		wg.Wait()
+		log.Info(ctx, "successfully handled metadata")
 	}()
 	return datasetURLChan
 }
 
-// TODO - logExtractedDatasetURLs is temporary.
-// The dataset url should be sent to the content-updated topic here in the future.
-// But for the time being, we are going to extract the urls and print them
-func (h *ReindexRequestedHandler) logExtractedDatasetURLs(ctx context.Context, wgDataset *sync.WaitGroup, cfg *config.Config, datasetURLChan chan string, reindexReqEvent *models.ReindexRequested) int {
+func (h *ReindexRequestedHandler) sendExtractedDatasetURLs(ctx context.Context, wgDataset *sync.WaitGroup, cfg *config.Config, datasetURLChan chan string, reindexReqEvent *models.ReindexRequested) int {
```

Comment:
no need to pass in the wg as it's not used in this function any more
```suggestion
func (h *ReindexRequestedHandler) sendExtractedDatasetURLs(ctx context.Context, cfg *config.Config, datasetURLChan chan string, reindexReqEvent *models.ReindexRequested) int {
```